chart/templates/secrets/redis-secrets.yaml

```diff
@@ -22,6 +22,7 @@
 # when we need them. We will always deploy them defensively to make the executor
 # update path actually work.
 
+{{- if or (eq .Values.executor "CeleryExecutor") (eq .Values.executor "CeleryKubernetesExecutor") }}
```

Comment:
why not adding this to the existing condition instead of adding a new one?
```
{{- if and (or (eq .Values.executor "CeleryExecutor") (eq .Values.executor "CeleryKubernetesExecutor")) .Values.redis.enabled (not .Values.redis.passwordSecretName) }}
```

Reply:
I though it would be more readable, wrap the entire file which would be more efficient (no further logic like password generation need to run) and more DRY, as I would need to repeat this condition twice if I were to merge it with the existing conditions.

1. https://github.com/apache/airflow/blob/484a7d0ca07ce594861a5989ec3f939756de0389/chart/templates/secrets/redis-secrets.yaml#L29
1. https://github.com/apache/airflow/blob/484a7d0ca07ce594861a5989ec3f939756de0389/chart/templates/secrets/redis-secrets.yaml#L53

Happy to edit my code if you think otherwise though?